src/Xamarin.Android.Build.Tasks/Tasks/Lint.cs

```diff
@@ -205,6 +205,26 @@ public override bool Execute ()
 			Log.LogDebugTaskItems ("  LibraryDirectories:", LibraryDirectories);
 			Log.LogDebugTaskItems ("  LibraryJars:", LibraryJars);
 
+			foreach (var issue in DisabledIssuesByVersion) {
+				if (lintToolVersion < issue.Value) {
+					Regex issueReplaceRegex = new Regex ($"{issue.Key}(,)?");
```

Comment:
Instead of `(,)?`, I think this should use `$@"\b{issue.Key}\b"` (word boundaries), so that e.g. `UnusedResourcesAreFun` isn't matched against `UnusedResources`. `\b` would also "match" `,`.

Reply:
We still need the `(,)?` because we want to remove a ending comma with the string replace. 

`Foo,Foo1,Foo2`

if we don't include the comma we end up with 

`Foo,,Foo2`

or 

`Foo,Foo1,`

both of which lint will complain about. That said adding the word boundaries should work too.